stage1/prepare-app/prepare-app.c

```diff
@@ -77,8 +77,8 @@ static int get_machine_name(char *out, int out_len) {
 	pgoto_if(close(fd) != 0,
 		_fail, "Error closing \"/etc/machine-id\"");
 	goto_if(snprintf(out, out_len,
-			"rkt-%.8s-%.4s-%.4s-%.4s-%.8s",
-			buf, buf+8, buf+12, buf+16, buf+20) >= out_len,
+			"rkt-%.8s-%.4s-%.4s-%.4s-%.12s",
+			buf, buf+8, buf+12, buf+16, buf+24) >= out_len,
```

Comment:
Is the change to `buf+24` correct? I think the offset remains the same, so it should stay at `buf+20`.